app/products/calls/components/current_call_bar/current_call_bar.tsx

```diff
@@ -249,7 +253,11 @@ const CurrentCallBar = ({
                     </View>
                     <View style={style.text}>
                         {talkingMessage}
-                        <Text style={style.channelAndTime}>
+                        <Text
+                            style={style.channelAndTime}
+                            numberOfLines={1}
+                            ellipsizeMode='middle'
+                        >
```

Comment:
It makes sense to have the speaking user be ellipsized-middle, but I don't know if it makes as much sense for the channel name. That might look better as a regular ellipses...  Imo.

Reply:
@cpoile, but we show call duration on the same line. That's the challenge. I felt duration is more important to retain completely than channel name.

Reply:
You mean we can't do `~Very very long ch... • 10:00` ?

Reply:
To do that, we'd have to add some logic to understand when it would wrap, or limit to a fixed number of characters.

Reply:
Probably not worth it. Looks good enough then, thanks!